### DISCUSSION TITLE
Poster glb model

### DISCUSSION THREAD

**Discussion by Jasonp8000** [google/model-viewer#2987] [Q&A] [answered]
Are we able to create a "poster" glb Model which acts similar to a poster image? 

Use case: 

Model viewer displays a glb model of a simple dining plate. However in the AR scene the model shows the full dining set which includes various plates, bowls and cups. 

Thanks everyone in advance

**Comment by barzorke** [comment, marked as answer]
You might need to load the "simple" model first, then after a delay or a clic, switch to the real model.... not sure if it is good felling enough

**Comment by samaneh-kazemi** [comment]
Hi @Jasonp8000, yes you can generate a poster using https://modelviewer.dev/editor/. Simply upload your glb model and download the scene, and you can find the poster image there. Then you can use that to lazy load the model

https://modelviewer.dev/docs/#entrydocs-loading-attributes-poster
